core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/ValidateHandler.java

```diff
@@ -31,7 +31,7 @@
  * The {@link uk.gov.gchq.gaffer.store.schema.Schema} is used to validate the elements.
  * The isSkipInvalidElements flag on {@link Validate} is used to determine what to do with invalid {@link Element}s.
  */
-public class ValidateHandler implements OutputOperationHandler<Validate, Iterable<? extends Element>> {
+public class ValidateHandler implements OutputOperationHandler<Validate, Iterable<? extends Element>>, OperationWithSchemaHandler<Validate, Iterable<? extends Element>> {
```

Comment:
What is the purpose of this? It looks like it worked without it before? An interface with one implementation in an anti-pattern.